proposals/0126-atf-additional-transports.md

```diff
@@ -33,27 +33,22 @@ Main reasons :
 ## Proposed solution
 
 Use real mobile device as a transport adapter.
-Create Mobile application as a part of ATF infrastructure. 
-Mobile application may use [SDL android library](https://github.com/smartdevicelink/sdl_android)
-for communicating with SDL and TCP connection for communication with ATF.
-
-Mobile application (Mobile transport adapter) should provide to ATF side such RPCs:
- - ConnectToSDL(transport)
- - RemoveConnection(connection_id)
- - GetListOfAvailableTransports()
- - SendData(connection_id, data)
- 
-ATF should be able to connect to multiple Mobile transport adapters and provide ability to use any of them to test engineer. 
-
-ATF should provide such API to test engineer: 
- - ConnectToSDL(device, connection type)
- - RemoveConnection(connection)
- - SendRawData(connection)
- - GetListOfAvailableDevices()
- - GetDeviceInfo(device)
- 
-_СonnectToSDL_ should return connection object.
-Connection object should provide ability to create sessions on it. Session interface should not be changed.  
+Create Android mobile application as a part of ATF infrastructure.
+
+Mobile application should use HTTP connection with REST protocol for control communication with ATF using next API:
+ - GetDeviceInfo - Provides device's information
```

Comment:
@aderiabin probably some details would be useful here for understanding what is a device info

Reply:
@AKalinich-Luxoft Details have been added. (see 5716e7b)